### ISSUE TITLE
Enable Consistent Outputs From TF/Pytorch Faster-RCNN's

### ISSUE THREAD

**Issue by lcadalzo** [Trusted-AI/adversarial-robustness-toolbox#655]
**Is your feature request related to a problem? Please describe.**
The outputs of ART's [TF](https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/1.4.1/art/estimators/object_detection/tensorflow_faster_rcnn.py#L371-L380) and [Pytorch](https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/1.4.1/art/estimators/object_detection/pytorch_faster_rcnn.py#L220-L223) Faster R-CNN's are of a different format. It would be very useful if I could call `.predict()` using either a Keras or Pytorch detector and return an output in a consistent format.

**Describe the solution you'd like**
Ideally the TF Faster R-CNN would (or could, i.e. if an optional kwarg was added) return an output in the form currently returned by the Pytorch Faster R-CNN. To be more specific: a list of dictionaries (one dictionary for each image) each containing the following keys: `boxes`, `scores`, `labels` as described in the lines highlighted in the Pytorch link above.

edit: another aspect of establishing consistency is the format of the predicted boxes (e.g. [xmin, ymin, xmax, ymax] and whether or not the boxes are normalized by image height/width).

**Comment by beat-buesser** [comment]
Hi @lcadalzo Thank you very much for raising this issue! I think this feature is going to be very useful for many users of ART. We'll include it in ART 1.4.2.
